### ISSUE TITLE
Checkout fails for 4100000000000019 and 4000000000000341

### ISSUE THREAD

**Issue by ashwinputhige** [peterkeen/payola#136]
In both the cases, I see 400 Bad request in the console.

In case of 4000000000000341, stripe returns 
```error: "Your card was declined."
guid: "j2224g"
status: "errored"```

In both the cases, after many attempts, it displays the "This seems to be taking too long...." message. Please let me know if you need more data.

**Comment by peterkeen** [comment]
What version are you using, and is this for one-time payments or
subscriptions? A fix went into v1.3.2 for this problem, but I may have
missed a case.

**Comment by ashwinputhige** [comment]
Currently testing one-time payments using Checkout button example in wiki.
Version: `* payola-payments (1.3.2)`

**Comment by peterkeen** [comment]
Hmm. I'll look into it later today. The relevant javascript is here:

https://github.com/peterkeen/payola/blob/master/app/assets/javascripts/payola/checkout_button.js#L63-L85

**Comment by nfm** [comment]
I think this issue may have been fixed by https://github.com/peterkeen/payola/pull/159 - previously, the AJAX error hander function was being called with data set to null, as we weren't parsing the response as JSON.

@ashwinputhige do you think this was what you were seeing? Is there a TypeError in your browser's developer tools (in Chrome this would be something like "Uncaught TypeError: Cannot read property 'status' of null") before the timeout error message is shown?

**Comment by ajw725** [comment]
just wanted to chime in here. i was having this problem with 1.3.2, but the 1.4.0 update seems to have fixed it.

**Comment by eliotsykes** [comment]
Closing as appears to be fixed by 1.4.0
